src/languages/php.js

```diff
@@ -438,6 +440,8 @@ export default function(hljs) {
     NAMED_ARGUMENT,
     LEFT_AND_RIGHT_SIDE_OF_DOUBLE_COLON,
     hljs.C_BLOCK_COMMENT_MODE,
+    // Attributes are single line and can neither contain C_LINE_COMMENT_MODE
+    // nor hljs.HASH_COMMENT_MODE
```

Comment:
This statement is invalid. Attributes are [multiline](https://onlinephp.io?s=s7EvyCjg5VKOjnFJLShKTU4sSU3RAPIVkvNzc1PzSni5ijPzklOtFJSMDIxMdA1MdQ3MlXi59PURCjRjFXi50krzkksy8_MUSlKLSzQ0Fap5uThTkzPyFdQT1a15uWp5uSAS1gA%2C&v=8.4.5%2C8.2.20)

Reply:
Thank you, I have removed the comment. I confess: I have never heard of PHP 8 attributes before :)